doc/themes/scikit-learn/static/img/scikit-learn-logo.svg

```diff
@@ -1044,7 +1044,6 @@
       Something went wrong with that request. Please try again.
     </div>
 
-    
+
   </body>
```

Comment:
This look weird changes?

Reply:
the change is the whitespace at the end of the line, it's done automatically with the git hooks we have.